### DISCUSSION TITLE
Welcome to discussions on contact.engineering!

### DISCUSSION THREAD

**Discussion by pastewka** [ContactEngineering/topobank#721] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mcrot** [comment]
Hi all, my name is Michael Röttger. I'm a physicist and software developer and the main developer of [TopoBank](https://github.com/ContactEngineering/TopoBank) the web app behind contact.engineering. Every now and then I also contribute to [SurfaceTopography](https://github.com/ContactEngineering/SurfaceTopography) which provides the scientific foundation for the web app.

Let us know if you've found a bug, have ideas or other comments regarding this web app.
Just open a new discussion or participate to an existing one.
